MozillaRustComponents/Sources/FocusRustComponentsWrapper/Generated/nimbus.swift

```diff
@@ -1735,12 +1735,11 @@ public struct AppContext {
     public var androidSdkVersion: String?
     public var debugTag: String?
     public var installationDate: Int64?
-    public var homeDirectory: String?
```

Comment:
We can't change files under `Generated/` since they will be overwritten by the next AS bump. These are directly generated from uniffi. If you want this to be removed you need to remove it upstream [here](https://github.com/mozilla/application-services/blob/930e6a7876d585b7033b7b734e0bc9ee08925b83/components/nimbus/src/stateful/matcher.rs#L54).